account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -575,15 +575,18 @@ def create_invoice_action_button(self):
         '''Workaround for a v10 bug: if I call create_invoice_action()
         directly from the button, I get the context in parsed_inv'''
         return self.create_invoice_action()
+    
+    def get_parsed_invoice(self):
```

Comment:
```suggestion
    def _get_parsed_invoice(self):
```

Reply:
I know, but I will keep it as is as this is how it was implemented on the latest version, will be easier to work in case of forward/backport related fixes

Reply:
thanks